proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,155 @@
+
+## Application
+
+Groups **must** fill
+https://docs.google.com/forms/d/1Fzm6HQt5RWyzbCSMmwj_8G78vlP7kXlT5sGjARXhjYM/edit?usp=sharing
+to apply for the two-day training course in December 2015.
+There is a cover letter sample at the end of this document.
+
+## Selection Criteria
+
+Groups will be selected based on:
+
+-   Group members were involved to SCF/DC,
+    e.g. members were learners, helpers or instructors in a workshop,
+    members have opened issues or pull request to one of SCF/DC lessons.
+
+-   Group members were involved to a group that organizes Software Carpentry-like workshops,
+    e.g. [The Hacker Within](https://thehackerwithin.github.com/),
+    Python and R local user groups.
+
+-   The group's plan for the follow-up workshop.
+
+Groups will not be selected based on:
+
+-   **Affiliation with a particular institution, lab, or university.**
+    All groups will be considered equally.
+
+-   **Academic qualification**
+    Faculty and students will be considered equally,
+    as will groups from different academic disciplines.
+
+## Selection
+
+Each member of the mentoring subcommittee
+will have one vote for each group slot for the Instructor Training round.
+Groups with the highest number of votes will be invited for the Instructor Training
+until all slots are allocated.
+
+## Group Confirmation Process
+
+Selected groups will have one week to confirm participation.
+If they fail to do so,
+their slot will be given to the next group in sequence.
+
+## Timeline
+
+-   2015-10-19: proposal approved.
+-   2015-10-20: blog post announcing the selection goes online.
+-   2015-11-04: group applications for two-day online class open.
+-   2015-11-18: group applications for two-day online class close.
+-   2015-11-23: extraordinary mentoring subcommittee meeting to vote on groups applications.
+-   2015-11-24: groups confirmation process begins.
+-   2015-12-03: groups confirmation process ends.
+-   2015-12-04: blog post announcing group selected goes online.
+-   2015-12-07: two-day online class begins.
+-   2015-12-08: two-day online class ends.
+
+## Application Sample
+
+Dear SCF Mentoring Subcommittee,
+
+we are writing to express our interested
+to attend the two-day training course in December 2015
+as one group composed by:
+
+-   NAME_OF_1ST_STUDENT,
+-   NAME_OF_2ND_STUDENT,
+-   NAME_OF_3RD_STUDENT (GITHUB_ID on GitHub),
+-   NAME_OF_4TH_STUDENT,
+-   NAME_OF_5TH_STUDENT and
+-   NAME_OF_6TH_STUDENT (GITHUB_ID on GitHub).
+
+Every member of our group attended SCF/DC workshops at least once.
+NAME_OF_1ST_STUDENT taught X at the last SCF/DC workshop organized at our university
+that was lead by INSTRUCTOR_NAME.
+At that same workshop NAME_OF_2ND_STUDENT and NAME_OF_3RD_STUDENT were helpers.
+
+NAME_OF_4TH_STUDENT is the organizer of the Python MeetUp meetings at our university.
+NAME_OF_5TH_STUDENT is the organizer of the R MeetUp meetings at our university.
```

Comment:
> Wow, this is all pretty intimidating - I don't think most groups will have this much activity between them. Will we scare people off?

I reduce the list to avoid scare people off.